bentoml/utils/__init__.py

```diff
@@ -54,3 +58,22 @@ def isidentifier(s):
     except AttributeError:
         # str#isidentifier is only available in python 3
         return re.match(r"[A-Za-z_][A-Za-z_0-9]*\Z", s) is not None
+
+
+def pb_to_json(message, output_format='obj'):
+    result = MessageToJson(message)
+    if output_format == 'string':
+        return result
+    return json.loads(result)
+
+
+def pb_to_yaml(message, output_format='obj'):
```

Comment:
I would prefer to just remove output_format option here and simply return yaml string - if you need to modify output yaml, might be better just modify the proto message before converting to yaml